README.md

```diff
@@ -0,0 +1,66 @@
+# Codacy Chart
+
+This is the top level chart to run Codacy in enterprise mode.
+With this chart, it is possible to run all the components from Codacy,
+plugs all the dependencies with a single command line, including Codacy,
+PostgreSQL, RabbitMQ, Minio, Ingress etc...
+
+## Work in progress
+
+This chart is still a **Work In Progress** and is not ready for general usage. Our docker images are currently private,
+you will not be able to run the chart by yourself. If you are
+interested in trying out codacy contact our support at support@codacy.com.
+
+## Goals
+
+Core goals for this project:
+
+1.  Easy to scale pods horizontally
+2.  Easy to deploy, upgrade, maintain
+3.  Wide support of cloud service providers
+4.  Use external data stores, such as Minio (S3) or Amazon RDS (Postgres), whenever possible
+5.  Users should be able to use their own certificates or use Let's Encrypt
+6.  Users should be able to use a load balancer or an ingess
+
+We will try to leverage standard Kubernetes features:
+
+*   ConfigMaps for managing configuration. These will then get mapped or passed to Docker containers
+
+## Helm Charts
+
+![Helm Chart Structure](./images/charts.png)
```

Comment:
Use the following link https://docs.google.com/drawings/d/1o7z3L8XnnNjHBOTWKHiIYUkBP3DDiogdUyxNdUfzyfY/export/png

And make the doc public

Reply:
I can't make things in our shared drive directory public. The option would be to move it to my drive but I could end up deleting it or something. I think ill just leave it like it is.